extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php

```diff
@@ -218,41 +218,6 @@ public function configureRobotsBuilder( RobotsTxt $robots, $shallow = false ) {
 			}
 		}
 
-		// Paranoid check to make sure language wikis return only their rules without calling other
-		// wikis recursively.
-		// TODO - remove the code below once robots are served by the robots-txt service
-		if ( !$wgRequest->getBool( 'shallow' ) && !$shallow ) {
```

Comment:
related to this check - I think the `shallow` url query parameter and the parameter of `configureRobotsBuilder` method can be removed as well